### PR TITLE
Add `-json` and `-t` flag for  `nomad acl token create` command

### PR DESCRIPTION
**Pull request by dttung2905**
Hi team,

This PR aims to add `-json` and `-t` flag for `nomad acl token create` command. This is related to issue #15894 

Example output:
- With `-json` flag
```
❯ nomad acl token create -type="management" -global=true \
  -name="Cluster A Replication Token" \
  -token="abcxyz" -ttl=10m -json
{
    "AccessorID": "83a61305-15c2-c78a-1c59-75c44afa7a13",
    "CreateIndex": 927,
    "CreateTime": "2023-02-04T09:39:33.710007669Z",
    "ExpirationTTL": 600000000000,
    "ExpirationTime": "2023-02-04T09:49:33.710007669Z",
    "Global": true,
    "ModifyIndex": 927,
    "Name": "Cluster A Replication Token",
    "Policies": null,
    "Roles": null,
    "SecretID": "6d23da6a-677d-c7b9-02ac-f1c41ad5f5d0",
    "Type": "management"
}

```
- With `-t` flag
```
❯ nomad acl token create -type="management" -global=true \
  -name="Cluster A Replication Token" \
  -token="abcxyz" -ttl=10m -t='{{ .ExpirationTTL }}'

10m0s

```

### CODE REVIEW

File: .changelog/16055.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+cli: Add -json and -t flag to nomad acl token create command
```
Comment (jrasell) [inline .changelog/16055.txt:2]: nit: wrap the "code" specific items
```suggestion
cli: Add `-json` and `-t` flag to `nomad acl token create` command
```

File: command/acl_token_create.go [hunk 2]
```diff
@@ -53,6 +53,12 @@ Create Options:
     Specifies the time-to-live of the created ACL token. This takes the form of
     a time duration such as "5m" and "1h". By default, tokens will be created
     without a TTL and therefore never expire.
+
+  -json
+    Output the ACL token information in a JSON format.
```
Comment (jrasell) [inline command/acl_token_create.go:58]:
```suggestion
    Output the ACL token information in JSON format.
```

File: website/content/docs/commands/acl/token/create.mdx [hunk 3]
```diff
@@ -44,6 +44,10 @@ The `acl token create` command requires no arguments.
   form of a time duration such as "5m" and "1h". By default, tokens will be
   created without a TTL and therefore never expire.
 
+- `-json`:Output the ACL token information in a JSON format.
```
Comment (jrasell) [inline website/content/docs/commands/acl/token/create.mdx:47]:
```suggestion
- `-json`:Output the ACL token information in JSON format.
```

**Review by jrasell — CHANGES_REQUESTED** [review]
Hi @dttung2905 and thanks for raising this; it's looking great. I've made a couple of very minor suggestions inline. I also wonder if it would be possible to update the tests added slightly, in particular when testing the JSON flag to check we have valid JSON returned in the output writer. Thanks again!

**Comment by dttung2905** [comment]
Hi @jrasell. Thanks for the super quick review. I didn't check the semantic of JSON output i.e. all KV pairs because I think its abit uncessary. Hence, I have only added a check for a proper JSON formatted output. Please let me know what you think on this :pray:

**Review by jrasell — APPROVED** [review]
Thanks for the updates @dttung2905 and LGTM! The unmarshal testing trick is something I have used elsewhere, so it's nice to see others see this as an approach to take. Thanks!

**Pull request merged**